### ISSUE TITLE
Commands list at netplan.io/design/#commands is missing some commands, includes invalid commands

### ISSUE THREAD

**Issue by fruviad** [canonical-web-and-design/netplan.io#135]
## Summary

https://netplan.io/design/#commands documents the following commands as existing:

- Generate
- Apply
- List
- Update
- Config

Some of those don't appear to be viable netplan commands.  A "Set" and a "Show" command may also be listed; it's unclear whether they're intended to be listed as commands but weren't formatted properly during the creation of the document.  They would also not be viable netplan commands.

Additionally, netplan is interpreting the commands in a case-sensitive manner (at least on Linux) and the documentation should list the working commands as all lowercase to reflect that fact.

My Ubuntu 20.04 install with netplan has the following commands available:

- help
- apply
- generate
- info
- ip
- try

Note that some of the commands in that list are missing from the command list on the netplan.io/design/#commands documentation.  (The "try" command, in particular, would be helpful to propagate to new users.)

## Process

View the documentation at https://netplan.io/design/#commands and compare those commands with the commands currently accepted by the software.  Run "netplan" without any options to see the commands currently accepted and documented.

## Current and expected result

Demonstration that the invalid commands will not work as documented (in lowercase or mixed case):

root@mojo:/etc/netplan# netplan list
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'list' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan List
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'List' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan Update
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'Update' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan update
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'update' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan Config
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'Config' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan config
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'config' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan Set
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'Set' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan set
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'set' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan Show
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'Show' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

root@mojo:/etc/netplan# netplan show
usage: /usr/sbin/netplan  [-h] [--debug]  ...
/usr/sbin/netplan : error: argument : invalid choice: 'show' (choose from 'help', 'apply', 'generate', 'info', 'ip', 'try')

**Comment by anthonydillon** [comment]
@powersj do you know the right person to provide these updates?

**Comment by pmahnke** [comment]
Might be @mitechie now?

**Comment by mitechie** [comment]
It's Matthieu's team (don't know his GH username) who runs netplan in foundations.

**Comment by anthonydillon** [comment]
Thanks @mitechie, would you mind taking a look please @mclemenceau.

**Comment by powersj** [comment]
Pinging @slyon as well

**Comment by mclemenceau** [comment]
Thanks @powersj, Discussed with @slyon, He will answer in detail tomorrow.

**Comment by slyon** [comment]
Thanks for bringing this up @fruviad Indeed some commands listed here are not available (never have been) and others are missing. I've created #151 to update it according to the current/latest netplan 0.101 release. This is also what you should have available on 20.04, if fully updated:
```
$ netplan help
usage: /usr/sbin/netplan  [-h] [--debug]  ...

Network configuration in YAML

optional arguments:
  -h, --help  show this help message and exit
  --debug     Enable debug messages

Available commands:
  
    help      Show this help message
    apply     Apply current netplan config to running system
    generate  Generate backend specific configuration files from /etc/netplan/*.yaml
    get       Get a setting by specifying a nested key like "ethernets.eth0.addresses", or "all"
    info      Show available features
    ip        Retrieve IP information from the system
    set       Add new setting by specifying a dotted key=value pair like ethernets.eth0.dhcp4=true
    try       Try to apply a new netplan config to running system, with automatic rollback
```

I've updated some of the wording according to what is written inside the "Commands" section of `index.html`. Feel free to adopt/improve the wording. (I've allowed maintainer write access to this feature branch.)
